src/cli/java/org/commcare/util/cli/ApplicationHost.java

```diff
@@ -84,7 +88,10 @@ public ApplicationHost(CommCareConfigEngine engine, PrototypeFactory prototypeFa
     }
 
     public void setRestoreToRemoteUser(String username, String password) {
-        this.mLocalUserCredentials = new String[]{username, password};
+        this.username = username;
+        this.password = password;
+        String domain = PropertyManager.instance().getSingularProperty("cc_user_domain");

```

Comment:
I really thought there was a more centralized location to do this addition in the code these days, is that not true? (I might have also only done that with a cli change that isn't merged yet)

Reply:
This is centralized as of https://github.com/dimagi/commcare-core/pull/718, updating